int32 deltaUmLeft
int32 deltaUmRight
uint16 mVBattery
uint32 timestamp
